### DISCUSSION TITLE
How to make compact bar top?

### DISCUSSION THREAD

**Discussion by Freed-Wu** [zellij-org/zellij#3605] [Q&A]
Just like tmux's `set status-bar top`?

**Comment by cristiand391** [comment]
haven't used tmux in a while now... you mean how to move the compact bar to the top of your screen?

**Comment by Lockszmith-GH** [comment]
First read [this](https://github.com/zellij-org/zellij/discussions/1701#discussioncomment-3517152)

Now, about the tab bar at the top, as well as the status bar at the bottom, they are both plugins, see [this](https://zellij.dev/documentation/plugin-aliases).

I haven't done any modifications on them, so I'm not sure, but look into these plugins and see if they have any configuratoin - if not, you might find better luck with [other plugins](https://github.com/zellij-org/awesome-zellij?tab=readme-ov-file#plugins)

**Comment by pezcore** [comment]
Use this layout file, which is just a copy of the built-in `compact.kdl` but with the pane order swapped to put the status bar (which I guess is technically pane?) at the top:

```kdl
layout {
    pane size=1 borderless=true {
        plugin location="compact-bar"
    }
    pane
}
```

Give this file a name with a `.kdl` suffix and copy it to the `~/.config/zellij/layouts` directory. 
Now you can name this layout in the [default_layout](https://zellij.dev/documentation/options.html#default_layout) section of the config
